### PR TITLE
feat: closing telescope after action is now optional

### PR DESCRIPTION
**Pull request by Jeansidharta**
This would resolve [this](https://github.com/AckslD/nvim-neoclip.lua/blob/6617f6cf8422be01eb5673e165ca20bc5c048085/lua/neoclip/telescope.lua#L61) and [this](https://github.com/AckslD/nvim-neoclip.lua/blob/6617f6cf8422be01eb5673e165ca20bc5c048085/lua/neoclip/telescope.lua#L77) todos.

It adds a new settings at `on_select`, `on_paste` and `on_replay` called `close_telescope` (defaults to true), which when set to true, will close telescope after the action. If set to false, the action will be executed while maintaining the telescope window open.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,13 +160,16 @@ use {
...
+  * `close_telescope`: if telescope should close whenever an item is selected.
 * `on_paste`:
   * `set_reg`: if the register should be populated when pressing the key to paste directly.
-  * `move_to_front`: if the entry should be set as the lastest in the list when pressing the key to paste directly.
+  * `move_to_front`: if the entry should be set to last in the list when pressing the key to paste directly.
+  * `close_telescope`: if telescope should close whenever a yank is pasted
 * `on_replay`:
   * `set_reg`: if the register should be populated when pressing the key to replay a recorded macro.
-  * `move_to_front`: if the entry should be set as the lastest in the list when pressing the key to replay a recorded macro.
+  * `move_to_front`: if the entry should be set to last in the list when pressing the key to replay a recorded macro.
```
Comment (Jeansidharta) [inline README.md:171]: I noticed the `move_to_front` sections of the README had some grammatical errors (I think I was sleepy when I wrote that). I took the liberty to fix them

**Comment by Jeansidharta** [comment]
Hey @AckslD, just following up on the last PR.

I found an even easier way to play actions on a buffer behind telescope. All I had to do was get the user's current buffer before opening Telescope, and pass it along to the actions, which would in turn apply their actions on the given buffer.

Therefore, [this](https://github.com/AckslD/nvim-neoclip.lua/blob/6617f6cf8422be01eb5673e165ca20bc5c048085/lua/neoclip/telescope.lua#L29-L45) function was not needed for these cases.

File: README.md (hunk 1 above)
Comment (AckslD) [reply README.md:171]: Great, I missed this as well :)

**Comment by AckslD** [comment]
Very nice @Jeansidharta!

**Pull request merged**